elastic/security/templates/composable/.logs-endpoint.action.responses.json

```diff
@@ -24,7 +24,8 @@
       ".logs-endpoint.action.responses@package",
       ".logs-endpoint.action.responses@custom",
       ".fleet_globals-1",
-      ".fleet_agent_id_verification-1"
+      ".fleet_agent_id_verification-1",
+      "track-shared-logsdb-mode"
```

Comment:
This are just my observations for security and logsdb:

- The index pattern doesn't match with logs-*, the pattern used here is: `.logs-endpoint.action.responses-*`
- The built in logging stack templates are not used here, which mean logsdb will never be enabled automatically.